pkg/forwarder/forwarder.go

```diff
@@ -62,7 +62,7 @@ const (
 	v1SeriesEndpoint       = "/api/v1/series"
 	v1CheckRunsEndpoint    = "/api/v1/check_run"
 	v1IntakeEndpoint       = "/intake/"
-	v1SketchSeriesEndpoint = "/api/v1/sketches"
+	v1SketchSeriesEndpoint = "/api/v1/sketches" //nolint unused for now
```

Comment:
What is the purpose of this comment?

Reply:
`// nolint` disable linting issue for that line. We don't use this variable for now but we want to keep it for future evolution